windows-service-guide.md

```diff
@@ -0,0 +1,76 @@
+# Cloud SQL Auth Proxy Windows Service Guide
+
+This document covers running the *Cloud SQL Auth Proxy* as service
+on the Windows operating system.
+
+It was originally built and tested using Go 1.20.2 on Windows Server 2019.
+
+## Install the Windows Service
+
+First, install the binary by:
+
+1. Create a new empty folder e.g. `C:\Program Files\cloud-sql-proxy`
+2. Copy the binary and helper batch files
```

Comment:
Should we link to the releases page and add a note about installing the 64-bit version?